picard/file_dilaog.py

```diff
@@ -0,0 +1,274 @@
+from PyQt5.QtGui import *
+from PyQt5.QtWidgets import *
+from PyQt5.QtCore import *
+
+import os,sys
+if os.name == 'nt':
+    import win32api, win32con
+from datetime import datetime
+
+def Is_hidden(file_temp):
+    if os.name== 'nt':
+        attribute = win32api.GetFileAttributes(file_temp)
+        return attribute & (win32con.FILE_ATTRIBUTE_HIDDEN | win32con.FILE_ATTRIBUTE_SYSTEM)
+    else:
+        return file_temp.startswith('.')
+
+class ImgWidget1(QLabel):
+    """
+    Class used to show picture on Table Row
+    """
+    def __init__(self, imagePath,parent=None):
+        super(ImgWidget1, self).__init__(parent)
+        pic = QPixmap(imagePath)
+        self.setPixmap(pic)
+
+class File_dialog(QDialog):
+    """
+    Main class used for the dialog
+    """
+    def __init__(self,wi,he,path,parent=None):
+        self.path = path
+        self.formats=["MP3","FLAC","OGG","M4A","WMA","WAV","MP4","AAC","flv"]
+        self.formats_lower=[]
+        for i in self.formats:self.formats_lower.append(i.lower())
+        self.formats_show=["MP3","FLAC","OGG","M4A","WMA","WAV","MP4","AAC","flv"]
```

Comment:
Why are there `formats` and `formats_show` ?
If you really need a label & a value, keep them associated using a list of objects, instead of multiple independent lists.